### PR TITLE
Add missing status 400 and 429 reponses for Delete visitors endpoint

### PR DESCRIPTION
**Pull request by JuroUhlar**

### CODE REVIEW

File: schemas/fingerprint-server-api.yaml [hunk 1]
```diff
@@ -719,6 +739,48 @@ components:
...
+          type: 'object'
+          additionalProperties: false
+          title: ErrorDeleteVisitsResponseError
+          properties:
+            code:
+              type: string
+              description: >
+                Error code:
+                * `RequestCannotBeParsed` - The visitor ID is invalid.
+                * `RequestCannotBeParsed` - The visitor ID is required.
```
Comment (JuroUhlar) [inline schemas/fingerprint-server-api.yaml:756]: Typo? 

```suggestion
```

Comment (JuroUhlar) [inline schemas/fingerprint-server-api.yaml:756]: Is this intentionally `RequestCannotBeParsed` twice?

Comment (realgisam) [reply schemas/fingerprint-server-api.yaml:756]: in both situation it's the same error code, with a different message
do you want to have a single message ?

Comment (JuroUhlar) [reply schemas/fingerprint-server-api.yaml:756]:
```suggestion
                * `RequestCannotBeParsed` - The visitor ID is missing or in the wrong format.
```

**Review by TheUnderScorer — DISMISSED** [review]

**Review by TheUnderScorer — APPROVED** [review]

**Pull request merged**